quickwit/quickwit-doc-mapper/src/default_doc_mapper/mapping_tree.rs

```diff
@@ -284,6 +487,10 @@ trait NumVal: Sized + FromStr + ToString + Into<TantivyValue> {
         }
     }
 
+    fn from_json(json_val: JsonValue, coerce: bool) -> Result<TantivyValue, String> {
+        Self::from_json_to_self(json_val, coerce).map(Self::into)
```

Comment:
```suggestion
        Self::from_json_to_self(json_val, coerce).map(TantivyValue::from)
```

Reply:
I'm not sure there is a clean way to do that. The trait is implemented for `Into<TantivyValue>`, so I can't call `from` (from implies into, but not the other way around). fwiw it was already a Self::into before, it just moved a bit